### PR TITLE
feat(next release): version bump

### PR DESCRIPTION
**Pull request by tfenster**

### CODE REVIEW

**Comment by tfenster** [comment]
@DavidFeldhoff I won't be able to take a look at #10 soon, #4 maybe is already resolved and #11 is long-term, so I would propose to create a new release. Do you agree?

**Comment by DavidFeldhoff** [comment]
TLDR: Go

I could pick up #10 the next days, #4 is definitely not solved as I saw it on Lucs machine, but I already wrote Luc personally and sent him a preview-vsix to try out some things to get more details. #11 might also be doable as we should be able to expand the relations of the work items and have to search for a parent there and in case there is one then we can display it. (so only showing the parent and not the whole hierarchy).

So I think 10 and 11 might be doable shortly. 4 might take longer, but nevertheless I think we don't have to hold back the things we already fixed until we fixed 10 and 11. That would take the "pressure" that we have to fix 10 and 11 directly the next days as I'm also a bit short the next days. Furthermore I also wouldn't have a problem with releases where we're only releasing one or two things, so "Go" :)

**Review by DavidFeldhoff — APPROVED** [review]
